cockpit/IPC/src/views/Configurator.tsx

```diff
@@ -130,26 +110,61 @@ export default function Configurator() {
   }
 
   const [activeItem, setActiveItem] = React.useState(Object.keys(schemas)[0]);
+  const [activeItemFilter, setActiveItemFilter] = React.useState<string | undefined>(undefined);
   const [form, setForm] = React.useState<React.JSX.Element>(<div />);
 
   const onSelect = (selectedItem: {
     groupId: string | number;
     itemId: string | number;
-    to: string;
-    event: React.FormEvent<HTMLInputElement>;
   }) => {
-    console.log(selectedItem.itemId);
-    setActiveItem(selectedItem.itemId as string);
-    setIsDrawerExpanded(false);
+    if (selectedItem.itemId) {
+      setActiveItem(selectedItem.itemId as string);
+      setIsDrawerExpanded(false);
+    } else if (selectedItem.groupId) {
+      setActiveItemFilter(selectedItem.groupId as string);
+    } else {
+      setActiveItemFilter(undefined);
+    }
   };
 
   function handleSubmit(data:any) {
     // eslint-disable-next-line no-param-reassign
     data = handleNullValue(data);
-    updateFormData(activeItem, data, setFormData, addAlert);
+    updateFormData(
+      activeItem, // Process name
+      activeItem, // Interface name
+      'Config', // Path
+      'config', // property
+      data, // Data
+      setFormData,
+      addAlert,
+    );
+  }
+
+  function getProcesses(): string[] {
+    // 1. Extract processes from schema keys
+    const extractedProcesses = Object.keys(schemas).map((schema) => {
+      const parts = schema.split('.');
+      return parts.slice(3, 5).join('.');
+    });
```

Comment:
We really should expose this information on the configuration interface s.t. you don't have to guess here.